lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -27,7 +27,8 @@ module Rails
       #   Time.current
       #   Time.zone.now
       #   Time.zone.parse('2015-03-02T19:05:37')
-      #   Time.zone.parse('2015-03-02T19:05:37Z') # Respect ISO 8601 format with timezone specifier.
```

Comment:
Was this an intentional removal? If so, what was the intention behind it?

Reply:
Sorry I wanted to explain the change in the PR body and forgot to do it

This example was added in PR #441 which specifically allows Time.parse when a timezone specifier is present so `Time.parse(...)` seems like the good syntax in that case

Reply:
Hm, this deleted example (`Time.zone.parse('2015-03-02T19:05:37Z')`) still seems like a good use case. Can you restore it instead of removing it?